source/jormungandr/jormungandr/scenarios/helper_classes/pt_journey.py

```diff
@@ -124,8 +126,48 @@ def _do_request(self):
         )
         return resp
 
+    def _do_isochrone_request(self):
+        logger = logging.getLogger(__name__)
+        fallback_durations_pool = (
```

Comment:
```python
fallback_durations_pool = self._orig_fallback_durtaions_pool or self._dest_fallback_durations_pool
assert fallback_durations_pool, abort(500, "blablabla")
```

Reply:
`or` can not work here since we get either a `None` or en empty dict.